File: guanabara-java/src/com/danillo/algoritmos/lista06/Exercicio63.java
package com.danillo.algoritmos.lista06;

import java.util.Scanner;

public class Exercicio63 {
    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);

        int soma = 0, menor = Integer.MAX_VALUE, par = 0, contador = 0;
        char continuar;

        do{
            System.out.println("Insira um número: ");
            int x = entrada.nextInt();

            if(x < menor){
                menor = x;
            }

            if(x % 2 == 0){
                par++;
            }

            soma = soma + x;
            contador++;

            System.out.println("Deseja continuar? (S ou N)");
            continuar = entrada.next().charAt(0);
        }while(continuar == 'S');

        float media = (float) soma/contador;

        System.out.println("Soma: " +soma+ "\nMenor valor: " +menor+ "\nMédia: " +media+ "\nValores pares: " +par);
    }
}

File: guanabara-python/src/lista06/exercicio63.py
i = 0
soma = 0
menor_numero = float('inf')
divisivel_por_2 = 0

while True:
    numero = int(input("Insira um número: "))

    soma = soma + numero

    if numero < menor_numero:
        menor_numero = numero

    if numero % 2 == 0:
        divisivel_por_2 = divisivel_por_2 + 1

    i = i + 1

    continuar = int(input("Deseja continuar: 1 - Sim | 2 - Não "))
    if continuar == 2:
        break

media = soma / i

print(f"Soma: {soma}.\nMenor número digitado: {menor_numero}.\nMédia: {media}.\nNúmeros pares: {divisivel_por_2}.")
